src/transforms/repair.jl

```diff
@@ -171,3 +173,60 @@ function _stretch10(g::Geometry)
   T = numtype(lentype(g))
   Stretch(ntuple(i -> one(T) + 10atol(T), embeddim(g)))
 end
+
+# ---------------
+# OPERATION (11)
+# ---------------
+
+function apply(::Repair{11}, poly::PolyArea)
+  r = rings(poly)
+  outer = r[begin]
+  inners = hasholes(poly) ? r[(begin + 1):end] : eltype(r)[]
+
+  # original orientations
+  oouter = orientation(outer)
+  oinners = orientation.(inners)
+  
+  # adjust orientation
+  oadjust(r, oold, onew) = oold == onew ? r : reverse(r)
+  outer = oadjust(outer, oouter, CCW)
+  inners = oadjust.(inners, oinners, CW)
+
+  PolyArea([outer; inners]), (oouter, oinners)
```

Comment:
try to follow the original implementation, and to avoid intermediate allocations